### DISCUSSION TITLE
[MediaCard] Support React nodes for the description

### DISCUSSION THREAD

**Discussion by mbaumbach** [Shopify/polaris-react#6963] [General] [answered]
## Feature request summary

Currently, the `MediaCard` component prop type definitions require that the description be a `string`. This feature request is to discuss opening that up to be `React.ReactNode` like the `title`.

## Rationale

We have a few use cases where we want more rich content in the `MediaCard` description, though it might be outside the desired scope of the component's intended use. I'm happy to submit a PR with the changes if this is something that sounds like a good idea.

---

🌟 Feature requests that are not yet planned will be closed. We then use the issue’s [:+1: upvotes](https://help.github.com/articles/about-conversations-on-github/) to track and set priorities. See the [contribution guidelines](https://github.com/Shopify/polaris-react/blob/master/.github/CONTRIBUTING.md) for more information.

**Comment by yurm04** [comment]
This seems like a broader discussion around when it's appropriate to use string vs. a react node in our APIs. Creating a GitHub  discussion to continue the conversation

**Comment by yurm04** [comment, marked as answer]
@mbaumbach I created a broader discussion https://github.com/Shopify/polaris/discussions/7005 on this topic. Would love to hear your thoughts on this. To keep things organized, I'm going to close this discussion and direct all related conversations to the new one linked. Thanks so much!
